src/App.js

```diff
@@ -1,15 +1,23 @@
 import "./App.css";
 import Navbar from "./Components/Navbar/Navbar";
 import Footer from "./Components/Footer/Footer";
+
+import { Routes, Route } from "react-router-dom";
+
+
+import Home from "./Components/Home/Home";
+import Cart from "./Components/Cart/Cart";
+import Products from "./Components/Products/Products";
```

Comment:
suggestion: you keep these files in pages folder, it would be great then your components and pages will not mix up, it will look clean, it will create a great folder structure

for reference see this [folder structure](https://discord.com/channels/914867137863118919/918560802590564422/951887611755106304)